spring-cloud-config-client/src/main/java/org/springframework/cloud/bootstrap/config/PropertySourceBootstrapConfiguration.java

```diff
@@ -143,12 +142,23 @@ public ConfigClientProperties configClientProperties() {
 		}
 
 		@Bean
+		@ConditionalOnMissingClass(name = {"org.springframework.cloud.config.server.EnvironmentRepository", "org.springframework.retry.RetryOperations"})
```

Comment:
Is there a better way to address what was discussed here: https://github.com/spring-cloud/spring-cloud-config/issues/72#issuecomment-75766916 ?

Sorry, if i missed something obvious, but basically I think we need to disable the `PropertySourceLocator` when the config server is bootstrapping.